src/main/webapp/sass/_core.scss

```diff
@@ -3271,10 +3271,10 @@ div.border {
 			color: inherit;
 		}
 		&.alert-info {
-			background: #DEEAF1;
-			border-left: 10px solid #5C94B9;
+			background: #D9ECF2;
```

Comment:
And make sure the alerts on Synapse.org align with design recommendations for this portal.
https://sagebionetworks.jira.com/browse/SWC-6160

Reply:
TODO: attempt to remove custom alert styles defined in SWC after update to the latest SRC (where the default FullWidthAlert styles should be correct).